lib/src/get_main.dart

```diff
@@ -29,54 +29,76 @@ class GetImpl implements GetService {
   RouteSettings settings;
   String defaultSeparator = "_";
 
-  ///Use to instead of Navigator.push, off instead of Navigator.pushReplacement,
-  ///offAll instead of Navigator.pushAndRemoveUntil. For named routes just add "named"
-  ///after them. Example: toNamed, offNamed, and AllNamed.
-  ///To return to the previous screen, use back().
-  ///No need to pass any context to Get, just put the name of the route inside
-  ///the parentheses and the magic will occur.
-
-  /// It replaces Navigator.push, but needs no context, and it doesn't have the Navigator.push
+  /// Pushes a new [page] to the stack
+  ///
+  /// It has the advantage of not needing context, so you can call from your business logic
+  ///
+  /// You can set a custom [transition], a transition [duration].
+  /// You can send any type of value to the other route in the [arguments].
+  /// If you're using the [Bindings] api, it's here that you use it
+  ///
+  /// It replaces Navigator.push and it doesn't have the Navigator.push
   /// routes rebuild bug present in Flutter. If for some strange reason you want the default behavior
```

Comment:
This bug has not existed since version 1.17.
I think this can now be removed. I don't know if I can change this in your branch, would you be able to remove this please for me?